### PR TITLE
Configure General State Tests

### PR DESCRIPTION
**Pull request by winsvega**
From this point we could remove old StateTests and switch to new GeneralStateTests
Fix transaction test time execution by defining mainnetwork test genesis

### CODE REVIEW

File: test/libethereum/State.cpp [hunk 1]
```diff
@@ -93,6 +93,8 @@ void doStateTests(json_spirit::mValue& _v, bool _fillin)
 }
 } }// Namespace Close
 
+//Disable OLD StateTests. Use GeneralStateTests at StateTests.cpp instead
+/*
```
Comment (chriseth) [inline test/libethereum/State.cpp]: Please use `//` to make it better visible that the tests are disabled. Why did you disable the tests? Does it make sense to delete them completely?

Comment (winsvega) [inline test/libethereum/State.cpp]: I've disabled it so it wont take execution time on windows build. 
Those tests are now implemented at GeneralStateTests so no need to run it twice.

Comment (chriseth) [inline test/libethereum/State.cpp]: But that means they can be removed, right?

File: test/libethereum/State.cpp [hunk 2]
```diff
@@ -93,506 +93,507 @@ void doStateTests(json_spirit::mValue& _v, bool _fillin)
...
-}
-
-BOOST_AUTO_TEST_CASE(userDefinedFileState)
-{
-	dev::test::userDefinedTest(dev::test::doStateTests);
-}
-
-BOOST_AUTO_TEST_SUITE_END()
-
+//Disable OLD StateTests. Use GeneralStateTests at StateTests.cpp instead
```
Comment (chfast) [inline test/libethereum/State.cpp]: Just delete it.

File: test/Stats.h [hunk 3]
```diff
@@ -21,6 +21,7 @@
 #include <vector>
 
 #include "TestHelper.h"
+//Execution stats (time/gas usage/OOG exceptions) for vm tests and tests that are using the vm.
```
Comment (chfast) [inline test/Stats.h]: What is this comment about?

File: libethashseal/genesis/frontier.cpp [hunk 4]
```diff
@@ -16,8 +16,8 @@
 */
 #include "../GenesisInfo.h"
 
-static dev::h256 const c_genesisStateRootFrontier("d7f8974fb5ac78d9ac099b9ad5018bedc2ce0a72dad1827a1709da30580f0544");
-static std::string const c_genesisInfoFrontier = std::string() +
+static dev::h256 const c_genesisStateRootMainNetwork("d7f8974fb5ac78d9ac099b9ad5018bedc2ce0a72dad1827a1709da30580f0544");
+static std::string const c_genesisInfoMainNetwork = std::string() +
```
Comment (chfast) [inline libethashseal/genesis/frontier.cpp]: Now MainNetwork means Frontier?

File: test/libethereum/TransactionTests.cpp [hunk 5]
```diff
@@ -48,7 +48,7 @@ void doTransactionTests(json_spirit::mValue& _v, bool _fillin)
 		eth::Network sealEngineNet;
 		BOOST_REQUIRE(o.count("blocknumber") > 0);
 		u256 transactionBlock = toInt(o["blocknumber"].get_str());
-		sealEngineNet = eth::Network::MainNetwork;
+		sealEngineNet = eth::Network::MainNetworkTest;
```
Comment (chfast) [inline test/libethereum/TransactionTests.cpp]: Please tell me you don't parse the JSON every iteration of the loop.

File: libethashseal/genesis/frontier.cpp (hunk 4 above)
Comment (winsvega) [inline libethashseal/genesis/frontier.cpp]: now mainNetwork means take c_geneisInfoMainNetwork as a genesis config

File: test/libethereum/TransactionTests.cpp (hunk 5 above)
Comment (winsvega) [inline test/libethereum/TransactionTests.cpp]: actually I do.

File: test/Stats.h (hunk 3 above)
Comment (winsvega) [inline test/Stats.h]: about Stats.h and what does it do.  There were complaints about code lacking comment coverage.

**Pull request merged**